src/components/OverviewSection/OverviewSection.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageBanner from '../PageBanner/PageBanner';
+import CategoryIndicator from '../CategoryIndicator/CategoryIndicator';
+import {
+  CATEGORIES,
+  STRINGS,
+} from '../../constants/index';
+import getColorVariant from '../../helpers/getColorVariant';
+import style from './OverviewSection.scss';
+
+const HERO_INDICATOR = 'population';
+
+const OverviewSection = (props) => {
+  const description = props.city
+    ? props.city.description
+    : STRINGS.OVERVIEW_DESCRIPTION;
+
+  const title = props.city
+    ? props.city.shortName
+    : STRINGS.OVERVIEW_TITLE;
+
+  return (
+    <div className={style.wrapper}>
+      <PageBanner
+        colorName="primary"
+        description={description}
+        indicator={HERO_INDICATOR}
+        title={title}
+        cities={props.cities}
+        city={props.city}
+      />
+
+      <div className={style.indicatorCards}>
+        {CATEGORIES.map(category => (
+          <div
+            key={category.id}
+            className={style.indicatorCardWrapper}
+            style={{
+              background: getColorVariant(category.colorName, '040'),
+            }}
+          >
+            <CategoryIndicator
+              className={style.categoryIndicator}
+              city={props.city}
+              cities={props.cities}
+              category={category}
+            />
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const cityPropShape = {
+  description: PropTypes.string.isRequired,
+  indices: PropTypes.object.isRequired,
+  shortName: PropTypes.string.isRequired,
+};
+
+OverviewSection.propTypes = {
```

Comment:
I feel a bit squeamish about this (and how it ripples into other components too). Not sure if I have a better solution though.

Reply:
This might be of interest: https://github.com/airbnb/prop-types
(Specifically: https://github.com/airbnb/prop-types/blob/master/src/mutuallyExclusiveProps.js )

Reply:
Yeah, I hear ya. But I think in practice there's nothing that could go wrong and make it into production without a dev-time warning, (if neither are passed in it fails loudly) so I don't think it's worth adding more complexity in the form of another package when it wouldn't have any material impact.

Reply:
I was thinking more of the case where _both_ are passed.

Reply:
Ah, gotcha. Yes then all sorts of unknown things would go wrong.